pkg/datapath/linux/routing/routing_test.go

```diff
@@ -137,19 +137,19 @@ func runFuncInNetNS(c *C, run func(), macAddr mac.MAC) {
 
 	currentNS, err := netns.Get()
 	c.Assert(err, IsNil)
-	c.Logf("Root network ns %v", currentNS.UniqueId())
+	c.Logf("[DEBUG] Root network ns %v", currentNS.UniqueId())
```

Comment:
It might be a bit weird to have these tags only for this specific unit tests. Maybe we should extend to the rest? Probably ok for this PR though.

Reply:
I was going to drop that commit after this PR reaches its goals. But if you think that's helpful, I can try to find a more sustainable, general solution for debug log msgs.

Reply:
Not sure it's worth it. Current log messages are a bit annoying to parse but adding this sort of additional tags would require a lot of small changes to all the tests.

Reply:
On second thought, I think keeping it will help slightly it debugging if the test flakes. If in the future, we don't like its verbosity, we'll revert.